README.md

```diff
@@ -21,9 +27,14 @@ the contracts, tests and deployment files and scripts. Docker configuration
 files are also created, for easy use of the aeternity blockchain network.
 
 ## Compile sophia contracts
+```
+aeproject compile
+```
 
-The **compile** command compiles sophia contract. Files should be with .aes
-file extension. Default directory is $projectDir/contracts.
+The **compile** command compiles Sophia contract. Files should be with .aes
+file extension. Default directory is $projectDir/contracts. The result of the compilation is the contract bytecode
```

Comment:
It's not necessary to be with .aes

Reply:
Are we sure that the compilation works with other file extension?

Reply:
Yes

Reply:
You might write something: "it's recommended to use .aes extension"